2024-09-12 12:34:49,927 WARNING:  * Debugger is active! [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:34:49,936 INFO:  * Debugger PIN: 701-334-296 [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:34:54,963 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:34:54,963 WARNING: Attempted to fetch processes without a selected agent [in D:\New-Dashboard\2\app.py:204]
2024-09-12 12:34:54,964 INFO: 127.0.0.1 - - [12/Sep/2024 12:34:54] "[31m[1mGET /api/processes HTTP/1.1[0m" 400 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:34:55,680 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:197]
2024-09-12 12:34:55,698 INFO: 127.0.0.1 - - [12/Sep/2024 12:34:55] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:34:56,151 INFO: 127.0.0.1 - - [12/Sep/2024 12:34:56] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:34:56,187 INFO: 127.0.0.1 - - [12/Sep/2024 12:34:56] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:34:56,281 INFO: 127.0.0.1 - - [12/Sep/2024 12:34:56] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:34:56,290 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:34:56,290 WARNING: Attempted to fetch processes without a selected agent [in D:\New-Dashboard\2\app.py:204]
2024-09-12 12:34:56,291 INFO: 127.0.0.1 - - [12/Sep/2024 12:34:56] "[31m[1mGET /api/processes HTTP/1.1[0m" 400 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:01,307 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:01,307 WARNING: Attempted to fetch processes without a selected agent [in D:\New-Dashboard\2\app.py:204]
2024-09-12 12:35:01,307 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:01] "[31m[1mGET /api/processes HTTP/1.1[0m" 400 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:06,306 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:06,306 WARNING: Attempted to fetch processes without a selected agent [in D:\New-Dashboard\2\app.py:204]
2024-09-12 12:35:06,307 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:06] "[31m[1mGET /api/processes HTTP/1.1[0m" 400 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:11,295 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:11,295 WARNING: Attempted to fetch processes without a selected agent [in D:\New-Dashboard\2\app.py:204]
2024-09-12 12:35:11,296 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:11] "[31m[1mGET /api/processes HTTP/1.1[0m" 400 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:16,134 INFO: Accessed agents page [in D:\New-Dashboard\2\app.py:66]
2024-09-12 12:35:16,306 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:16,306 WARNING: Attempted to fetch processes without a selected agent [in D:\New-Dashboard\2\app.py:204]
2024-09-12 12:35:16,306 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:16] "[31m[1mGET /api/processes HTTP/1.1[0m" 400 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:16,566 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:16] "GET /agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:16,614 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:16] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:16,617 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:16] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:16,618 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:16] "[36mGET /static/js/agents.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:16,639 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:274]
2024-09-12 12:35:16,639 WARNING: No agent currently selected [in D:\New-Dashboard\2\app.py:283]
2024-09-12 12:35:16,639 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:16] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:17,724 INFO: Selecting agent with ID: 3 [in D:\New-Dashboard\2\app.py:80]
2024-09-12 12:35:17,736 INFO: Agent selected: ubuntu (172.24.110.124) [in D:\New-Dashboard\2\app.py:92]
2024-09-12 12:35:17,737 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:17] "GET /select_agent/3 HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:18,808 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:274]
2024-09-12 12:35:18,809 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:18] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:20,744 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:197]
2024-09-12 12:35:20,745 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:20] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:20,797 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:20] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:20,797 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:20] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:20,799 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:20] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:20,820 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:20,823 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:35:21,622 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:197]
2024-09-12 12:35:21,624 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:21] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:21,686 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:21] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:21,689 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:21] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:21,690 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:21] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:21,715 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:21,716 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:35:22,876 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054A26C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:35:22,877 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:22] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:23,108 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:197]
2024-09-12 12:35:23,109 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:23] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:23,150 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:23] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:23,153 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:23] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:23,156 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:23] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:23,188 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:23,189 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:35:23,768 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054A2E40>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:35:23,769 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:23] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:25,245 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054E1580>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:35:25,245 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:25] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:28,189 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:28,190 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:35:30,209 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054CE270>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:35:30,210 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:30] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:33,190 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:33,193 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:35:35,249 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054CF6E0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:35:35,249 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:35] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:38,193 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:38,194 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:35:40,235 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054A0F80>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:35:40,235 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:40] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:43,192 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:43,193 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:35:45,216 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054A21B0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:35:45,217 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:45] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:48,192 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:48,193 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:35:50,227 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054CD850>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:35:50,228 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:50] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:53,186 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:53,187 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:35:55,255 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E05326B10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:35:55,256 INFO: 127.0.0.1 - - [12/Sep/2024 12:35:55] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:35:58,201 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:35:58,202 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:00,218 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054CF8F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:00,218 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:00] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:03,835 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:03,837 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:05,856 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E05473C20>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:05,857 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:05] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:08,833 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:08,834 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:10,866 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054E1D30>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:10,867 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:10] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:13,823 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:13,825 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:15,865 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054A10D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:15,866 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:15] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:18,834 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:18,836 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:20,873 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054CFA70>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:20,873 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:20] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:23,822 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:23,824 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:25,899 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054E2A80>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:25,899 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:25] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:28,837 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:28,840 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:30,882 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054CD8E0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:30,883 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:30] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:33,825 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:33,826 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:35,858 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054CE810>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:35,859 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:35] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:38,832 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:38,833 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:40,849 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054A1B50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:40,849 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:40] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:43,829 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:43,830 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:45,875 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054A2810>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:45,876 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:45] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:48,858 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:48,860 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:50,896 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054CDAF0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:50,897 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:50] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:53,831 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:53,833 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:36:55,849 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054CF3E0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:36:55,849 INFO: 127.0.0.1 - - [12/Sep/2024 12:36:55] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:36:58,827 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:36:58,829 DEBUG: Starting new HTTP connection (1): 172.24.110.124:80 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:37:00,875 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='172.24.110.124', port=80): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022E054CE300>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:214]
2024-09-12 12:37:00,875 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:00] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:07,302 INFO:  * Detected change in 'D:\\New-Dashboard\\2\\app.py', reloading [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:09,152 WARNING:  * Debugger is active! [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:09,162 INFO:  * Debugger PIN: 701-334-296 [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:29,829 INFO:  * Detected change in 'D:\\New-Dashboard\\2\\app.py', reloading [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:31,372 WARNING:  * Debugger is active! [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:31,382 INFO:  * Debugger PIN: 701-334-296 [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:36,827 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:37:36,827 WARNING: Attempted to fetch processes without a selected agent [in D:\New-Dashboard\2\app.py:205]
2024-09-12 12:37:36,828 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:36] "[31m[1mGET /api/processes HTTP/1.1[0m" 400 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:43,506 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:37:43,507 WARNING: Attempted to fetch processes without a selected agent [in D:\New-Dashboard\2\app.py:205]
2024-09-12 12:37:43,507 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:43] "[31m[1mGET /api/processes HTTP/1.1[0m" 400 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:44,943 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:37:44,960 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:44] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:45,412 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:45] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:45,466 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:45] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:45,508 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:45] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:45,519 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:37:45,519 WARNING: Attempted to fetch processes without a selected agent [in D:\New-Dashboard\2\app.py:205]
2024-09-12 12:37:45,519 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:45] "[31m[1mGET /api/processes HTTP/1.1[0m" 400 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:46,564 INFO: Accessed agents page [in D:\New-Dashboard\2\app.py:67]
2024-09-12 12:37:46,918 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:46] "GET /agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:46,969 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:46] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:46,970 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:46] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:46,971 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:46] "[36mGET /static/js/agents.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:47,002 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:275]
2024-09-12 12:37:47,002 WARNING: No agent currently selected [in D:\New-Dashboard\2\app.py:284]
2024-09-12 12:37:47,002 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:47] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:48,058 INFO: Selecting agent with ID: 3 [in D:\New-Dashboard\2\app.py:81]
2024-09-12 12:37:48,070 INFO: Agent selected: ubuntu (172.24.110.124) [in D:\New-Dashboard\2\app.py:93]
2024-09-12 12:37:48,071 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:48] "GET /select_agent/3 HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:49,350 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:275]
2024-09-12 12:37:49,351 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:49] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:50,374 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:37:50,374 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:50] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:50,429 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:50] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:50,433 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:50] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:50,436 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:50] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:50,458 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:37:50,463 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:37:50,488 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:37:50,490 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:50] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:37:55,468 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:37:55,469 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:37:55,484 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:37:55,486 INFO: 127.0.0.1 - - [12/Sep/2024 12:37:55] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:00,463 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:38:00,464 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:38:00,492 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:38:00,494 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:00] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:05,460 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:38:05,461 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:38:05,489 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:38:05,490 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:05] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:10,478 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:38:10,481 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:38:10,487 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:38:10,490 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:10] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:15,462 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:38:15,464 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:38:15,468 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:38:15,469 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:15] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:16,111 INFO: Accessed agents page [in D:\New-Dashboard\2\app.py:67]
2024-09-12 12:38:16,127 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:16] "GET /agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:16,175 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:16] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:16,177 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:16] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:16,178 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:16] "[36mGET /static/js/agents.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:16,211 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:275]
2024-09-12 12:38:16,212 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:16] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:21,901 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:38:21,902 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:21] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:21,955 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:21] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:21,956 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:21] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:21,958 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:21] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:21,978 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:38:21,980 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:38:21,985 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:38:21,986 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:21] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:25,463 INFO: Accessed users page [in D:\New-Dashboard\2\app.py:103]
2024-09-12 12:38:25,464 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:38:25,500 DEBUG: http://172.24.110.124:5000 "GET /users HTTP/11" 200 548 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:38:25,518 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:25] "GET /users HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:25,569 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:25] "[33mGET /static/js/users.js HTTP/1.1[0m" 404 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:25,572 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:25] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:25,575 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:25] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:32,041 INFO: Accessed applications page [in D:\New-Dashboard\2\app.py:109]
2024-09-12 12:38:32,042 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:38:32,227 DEBUG: http://172.24.110.124:5000 "GET /applications HTTP/11" 200 39147 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:38:32,275 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:32] "GET /applications HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:32,324 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:32] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:32,856 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:32] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:38:32,862 INFO: 127.0.0.1 - - [12/Sep/2024 12:38:32] "[33mGET /static/js/applications.js HTTP/1.1[0m" 404 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:39:39,677 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:39:39,678 INFO: 127.0.0.1 - - [12/Sep/2024 12:39:39] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:39:39,732 INFO: 127.0.0.1 - - [12/Sep/2024 12:39:39] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:39:39,733 INFO: 127.0.0.1 - - [12/Sep/2024 12:39:39] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:39:39,733 INFO: 127.0.0.1 - - [12/Sep/2024 12:39:39] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:39:39,755 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:39:39,756 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:39:39,779 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:39:39,781 INFO: 127.0.0.1 - - [12/Sep/2024 12:39:39] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:39:44,762 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:39:44,764 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:39:44,789 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:39:44,790 INFO: 127.0.0.1 - - [12/Sep/2024 12:39:44] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:39:49,765 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:39:49,766 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:39:49,793 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:39:49,795 INFO: 127.0.0.1 - - [12/Sep/2024 12:39:49] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:39:54,775 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:39:54,777 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:39:54,783 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:39:54,784 INFO: 127.0.0.1 - - [12/Sep/2024 12:39:54] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:39:59,761 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:39:59,764 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:39:59,789 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:39:59,791 INFO: 127.0.0.1 - - [12/Sep/2024 12:39:59] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:04,762 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:04,763 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:04,792 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:04,793 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:04] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:09,819 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:09,821 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:09,827 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:09,830 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:09] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:14,834 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:14,835 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:14,840 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:14,843 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:14] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:19,834 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:19,836 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:19,863 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:19,864 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:19] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:24,831 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:24,834 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:24,858 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:24,860 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:24] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:29,833 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:29,834 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:29,861 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:29,863 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:29] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:34,879 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:34,880 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:34,885 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:34,887 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:34] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:39,822 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:39,823 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:39,826 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:39,827 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:39] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:44,836 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:44,837 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:44,862 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:44,864 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:44] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:49,821 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:49,823 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:49,851 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:49,852 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:49] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:54,834 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:54,836 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:54,859 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:54,860 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:54] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:40:59,823 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:40:59,824 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:40:59,851 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:40:59,853 INFO: 127.0.0.1 - - [12/Sep/2024 12:40:59] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:41:04,833 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:41:04,834 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:41:04,859 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:41:04,861 INFO: 127.0.0.1 - - [12/Sep/2024 12:41:04] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:41:36,825 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:41:36,827 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:41:36,834 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:41:36,836 INFO: 127.0.0.1 - - [12/Sep/2024 12:41:36] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:42:36,825 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:42:36,826 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:42:36,853 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:42:36,854 INFO: 127.0.0.1 - - [12/Sep/2024 12:42:36] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:43:36,825 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:43:36,826 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:43:36,849 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:43:36,851 INFO: 127.0.0.1 - - [12/Sep/2024 12:43:36] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:43:38,406 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:43:38,410 INFO: 127.0.0.1 - - [12/Sep/2024 12:43:38] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:43:38,447 INFO: 127.0.0.1 - - [12/Sep/2024 12:43:38] "GET /static/css/style.css HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:43:38,447 INFO: 127.0.0.1 - - [12/Sep/2024 12:43:38] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:43:38,448 INFO: 127.0.0.1 - - [12/Sep/2024 12:43:38] "GET /static/js/processes.js HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:43:38,490 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:43:38,524 INFO: 127.0.0.1 - - [12/Sep/2024 12:43:38] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:43:43,140 INFO: 127.0.0.1 - - [12/Sep/2024 12:43:43] "[31m[1mPOST /select_agent/3 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:43:43,499 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:43:43,501 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:43:43,525 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:43:43,527 INFO: 127.0.0.1 - - [12/Sep/2024 12:43:43] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:43:48,487 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:43:48,488 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:43:48,492 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:43:48,493 INFO: 127.0.0.1 - - [12/Sep/2024 12:43:48] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:43:53,490 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:43:53,491 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:43:53,497 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:43:53,499 INFO: 127.0.0.1 - - [12/Sep/2024 12:43:53] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:43:58,530 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:43:58,532 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:43:58,537 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:43:58,539 INFO: 127.0.0.1 - - [12/Sep/2024 12:43:58] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:03,822 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:03,823 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:03,849 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:03,850 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:03] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:08,826 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:08,827 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:08,853 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:08,854 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:08] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:13,833 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:13,835 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:13,859 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:13,861 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:13] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:18,832 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:18,833 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:18,859 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:18,860 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:18] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:23,834 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:23,835 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:23,862 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:23,863 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:23] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:28,828 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:28,830 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:28,853 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:28,854 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:28] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:33,828 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:33,829 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:33,855 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:33,856 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:33] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:38,825 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:38,827 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:38,833 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:38,834 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:38] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:43,822 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:43,823 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:43,851 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:43,853 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:43] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:48,500 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:48,501 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:48,527 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:48,528 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:48] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:49,086 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:44:49,087 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:49] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:49,125 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:49] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:49,126 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:49] "GET /static/css/style.css HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:49,129 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:49] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:49,173 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:44:49,194 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:49] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:53,047 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:53] "[31m[1mPOST /select_agent/3 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:54,172 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:54,173 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:54,198 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:54,199 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:54] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:44:59,182 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:44:59,183 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:44:59,210 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:44:59,211 INFO: 127.0.0.1 - - [12/Sep/2024 12:44:59] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:02,355 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:02] "[31m[1mPOST /select_agent/2 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:04,183 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:45:04,184 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:45:04,210 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:45:04,211 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:04] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:08,261 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:08] "[31m[1mPOST /select_agent/1 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:09,175 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:45:09,176 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:45:09,202 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:45:09,204 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:09] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:11,041 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:45:11,041 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:11] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:11,076 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:11] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:11,079 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:11] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:11,081 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:11] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:11,124 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:45:11,141 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:11] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:13,355 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:13] "[31m[1mPOST /select_agent/1 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:15,784 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:15] "[31m[1mPOST /select_agent/2 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:16,129 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:45:16,131 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:45:16,149 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:45:16,151 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:16] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:21,128 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:45:21,129 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:45:21,148 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:45:21,149 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:21] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:22,076 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:22] "[31m[1mPOST /select_agent/1 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:24,112 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:45:24,112 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:24] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:24,145 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:24] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:24,149 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:24] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:24,151 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:24] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:24,193 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:45:24,211 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:24] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:27,009 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:27] "[31m[1mPOST /select_agent/2 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:29,190 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:45:29,191 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:45:29,206 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:45:29,208 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:29] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:34,191 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:45:34,192 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:45:34,217 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:45:34,218 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:34] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:38,268 INFO: Accessed agents page [in D:\New-Dashboard\2\app.py:67]
2024-09-12 12:45:38,283 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:38] "GET /agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:38,333 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:38] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:38,334 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:38] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:38,335 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:38] "[36mGET /static/js/agents.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:38,361 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:275]
2024-09-12 12:45:38,362 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:38] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:41,867 INFO: Managing agents - Method: DELETE [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:45:41,884 INFO: Removed agent with ID: 1 [in D:\New-Dashboard\2\app.py:163]
2024-09-12 12:45:41,885 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:41] "DELETE /api/agents?id=1 HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:43,093 INFO: Accessed agents page [in D:\New-Dashboard\2\app.py:67]
2024-09-12 12:45:43,108 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:43] "GET /agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:43,135 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:43] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:43,137 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:43] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:43,139 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:43] "[36mGET /static/js/agents.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:43,171 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:275]
2024-09-12 12:45:43,172 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:43] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:45,952 INFO: Managing agents - Method: DELETE [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:45:45,970 INFO: Removed agent with ID: 2 [in D:\New-Dashboard\2\app.py:163]
2024-09-12 12:45:45,970 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:45] "DELETE /api/agents?id=2 HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:47,228 INFO: Accessed agents page [in D:\New-Dashboard\2\app.py:67]
2024-09-12 12:45:47,244 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:47] "GET /agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:47,277 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:47] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:47,280 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:47] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:47,284 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:47] "[36mGET /static/js/agents.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:47,312 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:275]
2024-09-12 12:45:47,313 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:47] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:49,225 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:45:49,226 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:49] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:49,278 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:49] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:49,279 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:49] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:49,282 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:49] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:49,297 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:45:49,311 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:49] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:54,309 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:45:54,310 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:45:54,337 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:45:54,338 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:54] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:45:59,322 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:45:59,324 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:45:59,329 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:45:59,330 INFO: 127.0.0.1 - - [12/Sep/2024 12:45:59] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:00,026 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:00] "[31m[1mPOST /select_agent/3 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:04,831 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:04,832 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:04,836 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:04,838 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:04] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:09,829 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:09,830 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:09,858 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:09,860 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:09] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:14,830 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:14,832 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:14,839 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:14,841 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:14] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:19,833 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:19,834 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:19,838 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:19,839 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:19] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:24,823 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:24,825 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:24,853 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:24,855 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:24] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:29,827 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:29,828 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:29,855 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:29,858 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:29] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:34,824 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:34,825 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:34,851 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:34,853 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:34] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:39,832 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:39,833 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:39,862 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:39,865 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:39] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:44,830 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:44,833 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:44,840 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:44,842 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:44] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:49,826 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:49,827 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:49,854 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:49,855 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:49] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:54,828 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:54,830 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:54,856 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:54,857 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:54] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:46:59,826 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:46:59,827 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:46:59,850 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:46:59,852 INFO: 127.0.0.1 - - [12/Sep/2024 12:46:59] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:36,831 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:47:36,833 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:47:36,857 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:47:36,858 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:36] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:40,728 INFO: Fetching processes from agent [in D:\New-Dashboard\2\app.py:203]
2024-09-12 12:47:40,729 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:47:40,737 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:47:40,739 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:40] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:42,422 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:47:42,422 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:42] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:42,460 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:42] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:42,463 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:42] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:42,464 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:42] "GET /static/js/processes.js HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:42,502 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:47:42,517 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:42] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:43,088 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:47:43,089 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:43] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:43,126 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:43] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:43,127 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:43] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:43,128 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:43] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:43,162 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:47:43,175 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:43] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:43,843 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:47:43,844 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:43] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:43,879 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:43] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:43,882 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:43] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:43,882 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:43] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:43,920 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:47:43,935 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:43] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:44,178 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:47:44,180 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:44] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:44,223 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:44] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:44,227 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:44] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:44,228 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:44] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:47:44,267 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:47:44,309 INFO: 127.0.0.1 - - [12/Sep/2024 12:47:44] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:25,774 INFO:  * Detected change in 'D:\\New-Dashboard\\2\\app.py', reloading [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:27,442 WARNING:  * Debugger is active! [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:27,452 INFO:  * Debugger PIN: 701-334-296 [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:27,624 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:48:27,642 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:27] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:28,105 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:28] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:28,207 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:28] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:28,236 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:28] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:28,247 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:48:28,625 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:28] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:29,949 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:29] "[31m[1mPOST /select_agent/3 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:47,871 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:48:47,873 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:47] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:47,908 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:47] "GET /static/css/style.css HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:47,913 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:47] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:47,915 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:47] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:47,962 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:48:47,977 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:47] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:49,394 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:49] "[31m[1mPOST /select_agent/3 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:57,417 INFO:  * Detected change in 'D:\\New-Dashboard\\2\\app.py', reloading [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:58,952 WARNING:  * Debugger is active! [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:58,961 INFO:  * Debugger PIN: 701-334-296 [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:59,003 INFO: Accessed agents page [in D:\New-Dashboard\2\app.py:67]
2024-09-12 12:48:59,216 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:59] "GET /agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:59,710 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:59] "[36mGET /static/js/agents.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:59,732 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:59] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:59,798 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:59] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:48:59,825 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:278]
2024-09-12 12:48:59,825 WARNING: No agent currently selected [in D:\New-Dashboard\2\app.py:287]
2024-09-12 12:48:59,826 INFO: 127.0.0.1 - - [12/Sep/2024 12:48:59] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:49:01,501 INFO: Selecting agent with ID: 3 [in D:\New-Dashboard\2\app.py:81]
2024-09-12 12:49:01,514 INFO: Agent selected: ubuntu (172.24.110.124) [in D:\New-Dashboard\2\app.py:93]
2024-09-12 12:49:01,515 INFO: 127.0.0.1 - - [12/Sep/2024 12:49:01] "GET /select_agent/3 HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:49:02,671 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:278]
2024-09-12 12:49:02,671 INFO: 127.0.0.1 - - [12/Sep/2024 12:49:02] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:49:03,508 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:198]
2024-09-12 12:49:03,512 INFO: 127.0.0.1 - - [12/Sep/2024 12:49:03] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:49:03,556 INFO: 127.0.0.1 - - [12/Sep/2024 12:49:03] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:49:03,559 INFO: 127.0.0.1 - - [12/Sep/2024 12:49:03] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:49:03,561 INFO: 127.0.0.1 - - [12/Sep/2024 12:49:03] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:49:03,576 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:130]
2024-09-12 12:49:03,588 INFO: 127.0.0.1 - - [12/Sep/2024 12:49:03] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:49:04,745 INFO: 127.0.0.1 - - [12/Sep/2024 12:49:04] "[31m[1mPOST /select_agent/3 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:51:28,860 INFO:  * Detected change in 'D:\\New-Dashboard\\2\\app.py', reloading [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:51:30,599 WARNING:  * Debugger is active! [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:51:30,611 INFO:  * Debugger PIN: 701-334-296 [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:01,072 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:52:01,088 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:01] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:01,613 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:01] "GET /static/js/processes.js HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:01,632 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:01] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:01,654 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:01] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:01,680 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:134]
2024-09-12 12:52:02,033 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:02] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:03,410 INFO: Agent selected: ubuntu (172.24.110.124) [in D:\New-Dashboard\2\app.py:93]
2024-09-12 12:52:03,411 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:03] "POST /select_agent/3 HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:03,423 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:52:03,430 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:52:03,432 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:03] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:06,678 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:52:06,706 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:52:06,708 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:06] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:09,136 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:52:09,138 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:09] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:09,176 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:09] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:09,179 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:09] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:09,181 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:09] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:09,211 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:134]
2024-09-12 12:52:09,226 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:09] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:09,707 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:52:09,709 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:09] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:09,740 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:09] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:09,740 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:09] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:09,742 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:09] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:09,780 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:134]
2024-09-12 12:52:09,795 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:09] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:14,104 INFO: Accessed agents page [in D:\New-Dashboard\2\app.py:67]
2024-09-12 12:52:14,126 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:14] "GET /agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:14,187 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:14] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:14,189 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:14] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:14,189 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:14] "[36mGET /static/js/agents.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:14,216 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:282]
2024-09-12 12:52:14,216 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:14] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:26,730 INFO: Managing agents - Method: POST [in D:\New-Dashboard\2\app.py:134]
2024-09-12 12:52:26,747 INFO: Added new agent: mysql-server (192.168.1.28) [in D:\New-Dashboard\2\app.py:157]
2024-09-12 12:52:26,747 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:26] "[35m[1mPOST /api/agents HTTP/1.1[0m" 201 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:28,054 INFO: Accessed agents page [in D:\New-Dashboard\2\app.py:67]
2024-09-12 12:52:28,071 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:28] "GET /agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:28,109 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:28] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:28,112 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:28] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:28,114 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:28] "[36mGET /static/js/agents.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:28,139 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:282]
2024-09-12 12:52:28,140 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:28] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:32,320 INFO: Checking status of agent with ID: 4 [in D:\New-Dashboard\2\app.py:172]
2024-09-12 12:52:32,336 INFO: Updated status of agent 4 to Inactive [in D:\New-Dashboard\2\app.py:197]
2024-09-12 12:52:32,338 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:32] "GET /api/check_agent_status/4 HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:33,604 INFO: Accessed agents page [in D:\New-Dashboard\2\app.py:67]
2024-09-12 12:52:33,618 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:33] "GET /agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:33,650 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:33] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:33,652 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:33] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:33,653 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:33] "[36mGET /static/js/agents.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:33,685 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:282]
2024-09-12 12:52:33,686 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:33] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:35,410 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:52:35,411 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:35] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:35,470 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:35] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:35,473 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:35] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:35,474 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:35] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:35,496 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:134]
2024-09-12 12:52:35,523 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:35] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:37,297 INFO: Agent selected: mysql-server (192.168.1.28) [in D:\New-Dashboard\2\app.py:93]
2024-09-12 12:52:37,298 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:37] "POST /select_agent/4 HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:37,308 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:52:39,365 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F6C60>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:52:39,366 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:39] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:40,495 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:52:42,536 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F5460>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:52:42,538 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:42] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:45,501 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:52:47,545 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F6AB0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:52:47,545 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:47] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:50,827 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:52:52,869 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F51C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:52:52,869 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:52] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:52:55,820 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:52:57,876 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D70C560>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:52:57,876 INFO: 127.0.0.1 - - [12/Sep/2024 12:52:57] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:53:00,830 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:53:02,871 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D70D040>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:53:02,871 INFO: 127.0.0.1 - - [12/Sep/2024 12:53:02] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:53:05,822 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:53:07,875 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F6C60>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:53:07,875 INFO: 127.0.0.1 - - [12/Sep/2024 12:53:07] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:53:10,838 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:53:12,914 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F4E60>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:53:12,915 INFO: 127.0.0.1 - - [12/Sep/2024 12:53:12] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:53:15,822 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:53:17,868 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F5340>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:53:17,868 INFO: 127.0.0.1 - - [12/Sep/2024 12:53:17] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:53:20,920 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:53:22,961 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F4C50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:53:22,962 INFO: 127.0.0.1 - - [12/Sep/2024 12:53:22] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:53:25,825 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:53:27,896 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D70EEA0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:53:27,897 INFO: 127.0.0.1 - - [12/Sep/2024 12:53:27] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:53:30,837 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:53:32,855 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D70C200>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:53:32,856 INFO: 127.0.0.1 - - [12/Sep/2024 12:53:32] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:53:35,837 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:53:37,879 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D449730>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:53:37,880 INFO: 127.0.0.1 - - [12/Sep/2024 12:53:37] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:53:40,830 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:53:42,862 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F49B0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:53:42,862 INFO: 127.0.0.1 - - [12/Sep/2024 12:53:42] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:53:45,835 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:53:47,877 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F6330>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:53:47,877 INFO: 127.0.0.1 - - [12/Sep/2024 12:53:47] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:54:36,833 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:54:38,876 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F5610>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:54:38,877 INFO: 127.0.0.1 - - [12/Sep/2024 12:54:38] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:55:36,827 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:55:38,842 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D70F830>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:55:38,844 INFO: 127.0.0.1 - - [12/Sep/2024 12:55:38] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:03,255 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:56:04,520 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:56:04,521 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:04] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:04,560 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:04] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:04,565 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:04] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:04,566 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:04] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:04,602 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:134]
2024-09-12 12:56:04,621 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:04] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:05,302 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4C425490>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:56:05,302 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:05] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:06,470 INFO: Accessed users page [in D:\New-Dashboard\2\app.py:107]
2024-09-12 12:56:06,471 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:56:08,517 ERROR: Error communicating with agent: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /users (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F5280>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:56]
2024-09-12 12:56:08,525 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:08] "[35m[1mGET /users HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:08,577 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:08] "[36mGET /users?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:08,577 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:08] "[36mGET /users?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:08,624 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:08] "GET /users?__debugger__=yes&cmd=resource&f=console.png&s=JDAhQkOA6bj9znUu3WPW HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:08,692 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:08] "[36mGET /users?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:13,044 INFO: Accessed applications page [in D:\New-Dashboard\2\app.py:113]
2024-09-12 12:56:13,046 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:56:15,112 ERROR: Error communicating with agent: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /applications (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F72C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:56]
2024-09-12 12:56:15,119 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:15] "[35m[1mGET /applications HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:15,168 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:15] "[36mGET /applications?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:15,170 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:15] "[36mGET /applications?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:15,189 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:15] "GET /applications?__debugger__=yes&cmd=resource&f=console.png&s=JDAhQkOA6bj9znUu3WPW HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:15,241 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:15] "GET /applications?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:17,420 INFO: Accessed agents page [in D:\New-Dashboard\2\app.py:67]
2024-09-12 12:56:17,434 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:17] "GET /agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:17,480 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:17] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:17,481 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:17] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:17,482 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:17] "[36mGET /static/js/agents.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:17,508 INFO: Fetching selected agent information [in D:\New-Dashboard\2\app.py:282]
2024-09-12 12:56:17,509 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:17] "GET /api/selected_agent HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:19,512 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:19] "[31m[1mGET /select_agent/3 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:21,917 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:21] "[31m[1mGET /select_agent/3 HTTP/1.1[0m" 405 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:23,966 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:56:23,967 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:23] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:24,012 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:24] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:24,012 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:24] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:24,013 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:24] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:24,038 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:134]
2024-09-12 12:56:24,061 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:24] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:25,994 INFO: Agent selected: ubuntu (172.24.110.124) [in D:\New-Dashboard\2\app.py:93]
2024-09-12 12:56:25,994 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:25] "POST /select_agent/3 HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:26,005 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:56:26,020 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:56:26,021 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:26] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:26,913 INFO: Accessed users page [in D:\New-Dashboard\2\app.py:107]
2024-09-12 12:56:26,915 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:56:26,941 DEBUG: http://172.24.110.124:5000 "GET /users HTTP/11" 200 548 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:56:26,957 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:26] "GET /users HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:27,010 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:27] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:27,013 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:27] "[33mGET /static/js/users.js HTTP/1.1[0m" 404 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:27,014 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:27] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:30,446 INFO: Accessed applications page [in D:\New-Dashboard\2\app.py:113]
2024-09-12 12:56:30,448 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:56:30,521 DEBUG: http://172.24.110.124:5000 "GET /applications HTTP/11" 200 39147 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:56:30,569 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:30] "GET /applications HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:30,613 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:30] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:31,093 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:31] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:31,096 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:31] "[33mGET /static/js/applications.js HTTP/1.1[0m" 404 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:31,649 INFO: Accessed applications page [in D:\New-Dashboard\2\app.py:113]
2024-09-12 12:56:31,650 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:56:31,694 DEBUG: http://172.24.110.124:5000 "GET /applications HTTP/11" 200 39147 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:56:31,738 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:31] "GET /applications HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:31,773 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:31] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:32,363 INFO: Accessed users page [in D:\New-Dashboard\2\app.py:107]
2024-09-12 12:56:32,366 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:56:32,375 DEBUG: http://172.24.110.124:5000 "GET /users HTTP/11" 200 548 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:56:32,388 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:32] "GET /users HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:32,446 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:32] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:32,450 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:32] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:32,453 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:32] "[33mGET /static/js/users.js HTTP/1.1[0m" 404 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:57,548 INFO: Accessed users page [in D:\New-Dashboard\2\app.py:107]
2024-09-12 12:56:57,549 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:56:57,572 DEBUG: http://172.24.110.124:5000 "GET /users HTTP/11" 200 883 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:56:57,585 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:57] "GET /users HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:57,614 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:57] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:57,618 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:57] "[33mGET /static/js/users.js HTTP/1.1[0m" 404 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:56:57,619 INFO: 127.0.0.1 - - [12/Sep/2024 12:56:57] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:20,070 INFO: Accessed processes page [in D:\New-Dashboard\2\app.py:202]
2024-09-12 12:59:20,071 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:20] "GET /processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:20,127 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:20] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:20,131 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:20] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:20,132 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:20] "[36mGET /static/js/processes.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:20,175 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:134]
2024-09-12 12:59:20,199 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:20] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:22,692 INFO: Agent selected: mysql-server (192.168.1.28) [in D:\New-Dashboard\2\app.py:93]
2024-09-12 12:59:22,692 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:22] "POST /select_agent/4 HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:22,703 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:59:24,778 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D70CC20>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:59:24,778 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:24] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:25,173 DEBUG: Starting new HTTP connection (1): 192.168.1.28:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:59:25,373 INFO: Agent selected: ubuntu (172.24.110.124) [in D:\New-Dashboard\2\app.py:93]
2024-09-12 12:59:25,373 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:25] "POST /select_agent/3 HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:27,213 ERROR: Error communicating with agent for processes: HTTPConnectionPool(host='192.168.1.28', port=5000): Max retries exceeded with url: /processes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020E4D6F68D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')) [in D:\New-Dashboard\2\app.py:222]
2024-09-12 12:59:27,215 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:27] "[35m[1mGET /api/processes HTTP/1.1[0m" 500 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:27,220 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:59:27,247 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:59:27,247 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:27] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:30,167 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:59:30,193 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:59:30,194 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:30] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:32,193 INFO: Accessed home page [in D:\New-Dashboard\2\app.py:62]
2024-09-12 12:59:32,198 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:32] "GET / HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:32,255 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:32] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:32,258 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:32] "[36mGET /static/js/main.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:32,260 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:32] "[36mGET /static/js/dashboard.js HTTP/1.1[0m" 304 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:32,301 INFO: Managing agents - Method: GET [in D:\New-Dashboard\2\app.py:134]
2024-09-12 12:59:32,304 INFO: Managing users - Method: GET [in D:\New-Dashboard\2\app.py:227]
2024-09-12 12:59:32,305 INFO: Fetching applications from agent [in D:\New-Dashboard\2\app.py:237]
2024-09-12 12:59:32,307 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:59:32,308 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:59:32,311 DEBUG: Starting new HTTP connection (1): 172.24.110.124:5000 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:243]
2024-09-12 12:59:32,316 DEBUG: http://172.24.110.124:5000 "GET /users HTTP/11" 200 883 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:59:32,326 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:32] "GET /api/agents HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:32,328 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:32] "GET /api/users HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:32,338 DEBUG: http://172.24.110.124:5000 "GET /processes HTTP/11" 200 3074 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:59:32,340 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:32] "GET /api/processes HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
2024-09-12 12:59:32,400 DEBUG: http://172.24.110.124:5000 "GET /applications HTTP/11" 200 39147 [in D:\New-Dashboard\2\alpha\Lib\site-packages\urllib3\connectionpool.py:546]
2024-09-12 12:59:32,406 INFO: 127.0.0.1 - - [12/Sep/2024 12:59:32] "GET /api/applications HTTP/1.1" 200 - [in D:\New-Dashboard\2\alpha\Lib\site-packages\werkzeug\_internal.py:97]
